src/components/LikeCoinButtonWidget/LikeCoinButtonWidget.stories.js

```diff
@@ -167,15 +177,43 @@ const Controlled = ({
   `,
 });
 
-export const Case1 = Controlled();
+export const Liker = Controlled();
+
+export const LikerCanSuperLike = Controlled({
+  isSuperLikeEnabled: true,
+});
 
-export const Case2 = Controlled({ cooldown: 0.8 });
+export const LikerHasSuperLiked = Controlled({
+  count: 5,
+  isSuperLikeEnabled: true,
+  hasSuperLiked: true,
+  cooldown: 1,
+});
 
-export const Case3 = Controlled({
+export const LikerHadSuperLiked = Controlled({
   count: 5,
+  isSuperLikeEnabled: true,
   hasSuperLiked: true,
 });
 
-export const SuperLikeDisabled = Controlled({
-  isSuperLikeEnabled: false,
+export const Creator = Controlled({
+  isCreator: true,
+});
+
+export const CreatorCanSuperLike = Controlled({
+  isCreator: true,
+  isSuperLikeEnabled: true,
+});
+
+export const CreatorHasSuperLiked = Controlled({
+  isCreator: true,
+  isSuperLikeEnabled: true,
+  hasSuperLiked: true,
+  cooldown: 1,
+});
+
+export const CreatorHadSuperLiked = Controlled({
```

Comment:
The names `CreatorHasSuperLiked` and `CreatorHadSuperLiked` are quite confusing
(same for `LikerHasSuperLiked` and `LikerHadSuperLiked`)

Reply:
`has` mean the liker has Super Liked, and waiting for CD
`had` mean the liker had Super Liked once, he/she can Super Like again (CD has finished)

Can u suggest another name for the stories?

Reply:
`CreatorSuperLikeCountingDown` and `CreatorWithSuperLikeHistory`? But seems too long

Reply:
CoolDowning and CooldownReady?

Reply:
Maybe I add descriptions instead of changing the names later